internal/controller/vrg_recipe.go

```diff
@@ -79,44 +79,35 @@ func GetPVCSelector(ctx context.Context, reader client.Reader, vrg ramen.VolumeR
 	ramenConfig ramen.RamenConfig,
 	log logr.Logger,
 ) (PvcSelector, error) {
-	var recipeElements RecipeElements
+	recipeElements, err := RecipeElementsGet(ctx, reader, vrg, ramenConfig, log)
+	if err != nil {
+		return recipeElements.PvcSelector, err
```

Comment:
(nit) This line is a bit confusing. It references returning `recipeElements.PvcSelector` on an error, but since `filterPVC()` skips `PvcSelector` in case of an error, I suggest returning `PvcSelector{}` instead. This would improve readability.

Reply:
I merged this PR and fixed this nit in the next PR #1738